lib/oli_web/components/delivery/layouts.ex

```diff
@@ -379,7 +382,7 @@ defmodule OliWeb.Components.Delivery.Layouts do
         </div>
       </nav>
       <nav
-        id="mobile-workspace-nav-menu"
+        id="mobile-nav-menu"
```

Comment:
This was the root cause of the issue. However, more broadly because the menu button and this element and the menu button that triggers it are rendered separately, there was no guarantee that this element actually exists on the page and therefore would be broken on certain pages.

Refactoring the header to accept a `sidebar_enabled` attribute that is more in sync with the logic that renders the sidebar will help guarantee that the header hamburger menu button is only rendered when the supporting sidebar nav item is rendered as well.